Bejebeje.DataAccess/Migrations/20190511151840_InitialCreate.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Bejebeje.DataAccess.Migrations
+{
+    public partial class InitialCreate : Migration
```

Comment:
Not sure why you are doing this, no Web Expert, but isn't the power of EF to auto-map Tables, Entities, and auto create Migrations? why do you need to manually name and create all this??